feature-pack/client/src/main/resources/modules/io/netty/slot/module.xml

```diff
@@ -17,5 +17,7 @@
     <dependencies>
         <module name="javax.api"/>
         <module name="org.javassist" optional="true"/>
+        <module name="org.apache.log4j"/>
```

Comment:
Isn't this module log4j 1.x? I think Netty uses log4j 2.x.

Reply:
WildFly comes with neither. It has a JBoss Logging shim that pretends to be log4j [12]

Reply:
Ah ok